examples/horovod/tensorflow-mnist.yaml

```diff
@@ -15,7 +15,6 @@ spec:
             name: mpi-launcher
             command:
             - mpirun
-            args:
```

Comment:
Do we need to remove it?

Reply:
No, I just messed it up when reverting. Also, using a the v1 api works just fine. the apiserver does the conversion for us.